ai-ml/bionemo/eks.tf

```diff
@@ -109,12 +109,12 @@ module "eks" {
       )
 
       ami_type            = "AL2_x86_64_GPU"
-      ami_release_version = "1.29.0-20240213"
+      ami_release_version = "1.29.6-20240828"
```

Comment:
We don't have to specify the `ami_release_version` verison. Module will always get the latest AMI for the ami_type

Reply:
@vara-bonthu updated to remove the ami_release_version.

I also fixed the pre-commit failures.